tests/repo-test.js

```diff
@@ -0,0 +1,69 @@
+/* globals describe, before, after, it*/
+
+// var assert = require('assert')
+var expect = require('chai').expect
+var ncp = require('ncp').ncp
+var rimraf = require('rimraf')
+
+var IPFSRepo = require('./../src')
+
+describe('IPFS Repo Tests', function () {
+  var repo
+  var testRepoPath = __dirname + '/test-repo'
+  var date = Date.now().toString()
+  var repoPath = testRepoPath + date
+
+  before(function (done) {
+    ncp(testRepoPath, repoPath, function (err) {
+      if (err) {
+        expect(err).to.equal(null)
+      }
+      done()
+    })
+  })
+
+  after(function (done) {
+    rimraf(repoPath, function (err) {
+      if (err) {
+        expect(err).to.equal(null)
+      }
+      done()
+    })
+  })
+
+  it('check if Repo exists', function (done) {
+    repo = new IPFSRepo(repoPath)
+    expect(repo.exists()).to.equal(true)
+    done()
+  })
+
+  it('load the Repo', function (done) {
+    repo.load()
+    done()
+  })
+
+  it('init another Repo', function (done) { done() })
+
+  describe('api', function () {})
+  describe('config', function () {})
+  describe('version', function () {
+    it('get version', function (done) {
+      var readStream = repo.version().read()
+      var buffer = ''
+      readStream.on('data', function (chunk) {
```

Comment:
It would be nice to have a read + readSync API on top of the blob-store for things like reading one value, avoiding to have to write all of the stream stuff over and over again